apps/backend/src/assets/grpc/cacic/m2m/account_manager/v1/account-manager-m2m.proto

```
@@ -0,0 +1,90 @@
+syntax = "proto3";
+
+package cacic.m2m.account_manager.v1;
```

Buf (1.71.0)

[error] Files with package "cacic.m2m.account_manager.v1" must be within a directory "cacic/m2m/account_manager/v1" relative to root but were in directory "apps/backend/src/assets/grpc".

(PACKAGE_DIRECTORY_MATCH)